quickstart.py

```diff
@@ -309,6 +340,196 @@ def sendGetRequest(port):
   except urllib2.URLError:
     return None
 
+def stopDocker(config, forceStop = False):
+  """ Returns |False| on error. If Docker is not configured for restart and
+  |forceStop == False|, it will not be stopped and |True| (success) will be
+  returned
+  """
+  if not dockerRunning(config):
+    print "Docker already stopped."
+    return True
+
+  if not config["Docker"]["restart"] and not forceStop:
+    print "Not stopping Docker because it is not configured for restart"
+    return True
+
+  dockerApi = getDockerApi(config)
+  try:
+    dockerApi.stop(container = DOCKER_CONTAINER_NAME)
+  except:
+    print "Failed to stop Docker"
+    return False
```

Comment:
This is a bare try/except and it's not great. The print statement doesn't tell you anything about the exception. For example, this could be dying because you spelled `DOCKER` wrong in the variable value.

Given that you don't know what you want to catch here, it's better for your sanity to do something like this:

```
try:
    dockerApi.stop(container=DOCKER_CONTAINER_NAME)
except Exception as exc:
    print 'Failed to stop Docker:', exc
    return False
```